custom/icds_reports/management/commands/backfill_child_health_monthly.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+from datetime import date
+
+from dateutil.rrule import MONTHLY, rrule
+from django.core.management.base import BaseCommand
+from django.db import connections
+
+from corehq.sql_db.connections import get_icds_ucr_db_alias
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        start_date = date(2017, 3, 1)
+        dates = [dt for dt in rrule(MONTHLY, dtstart=start_date, until=date(2019, 2, 1))]
+        sql = """
+        UPDATE {tablename} monthly SET
+            supervisor_id = awc_loc.supervisor_id
+        FROM awc_location awc_loc
+        WHERE awc_loc.awc_id = monthly.awc_id AND monthly.supervisor_id IS NULL
+        """
+
+        for date_ in dates:
```

Comment:
split this up by month so that it doesn't lock the entire table, only one child table at at ime